parquet/src/main/java/org/apache/iceberg/parquet/ParquetBloomRowGroupFilter.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.parquet;
```

Comment:
Is it possible to include the unit test in this PR? I think all you'd need to do is configure the bloom filter settings using the Parquet settings in a Hadoop Configuration rather than through the Iceberg write settings.

Reply:
I tried to config using  `ParquetOutputFormat.BLOOM_FILTER_ENABLED` by replacing this [line](
https://github.com/apache/iceberg/blob/1fabb2db59d6db33609f549024c4371873ad3b10/parquet/src/test/java/org/apache/iceberg/parquet/TestBloomRowGroupFilter.java#L201) with `.set(ParquetOutputFormat.BLOOM_FILTER_ENABLED + "#_id", "true") `, but it doesn't work.

Seems Iceberg only honors the Iceberg's properties (which are set by `Context` at [here](https://github.com/apache/iceberg/blob/master/parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java#L233)), but it doesn't really take the properties set by Parquet.

Reply:
I think that properties will be passed through to the Hadoop `Configuration` automatically. Is that no longer true?

Reply:
Should I set the Parquet properties to `encodingPropsBuilder`? If the same properties is also set by iceberg properties, I will reset to overwrite.

Reply:
I see. Is it possible to get some tests working without the write-side changes? Maybe write a Parquet file directly and use name mapping? If not then let's try to make the minimal write-side changes to get the test in.

Reply:
I made some minimal write-side changes to get the test in. Hope this is OK.